lightningd/bitcoind.h

```diff
@@ -38,7 +38,7 @@ struct bitcoind {
 	struct lightningd *ld;
 
 	/* Are we currently running a bitcoind request (it's ratelimited) */
-	bool req_running;
+	struct bitcoin_cli *req_running;
```

Comment:
The name `req_running` still suggests that it is boolean. Shall we change it to `current` or something similar?

Reply:
Ack, will rebase w that change...